jax/lax_linalg.py

```diff
@@ -847,9 +854,9 @@ def _qr_cpu_gpu_translation_rule(geqrf_impl, orgqr_impl, c, operand,
 # Singular value decomposition
 
 def svd_impl(operand, full_matrices, compute_uv):
-  s, u, vt = xla.apply_primitive(svd_p, operand, full_matrices=full_matrices,
-                                 compute_uv=compute_uv)
-  return s, u, vt
+  result = xla.apply_primitive(svd_p, operand, full_matrices=full_matrices,
```

Comment:
(In the past, unpacking this sequence was actually necessary, but I don't think it is any more.)

Can we just `return xla.apply_primitive ...`